_pages/business-units/18f/leadership-resources/leadership-selection.md

```diff
@@ -0,0 +1,66 @@
+---
+title: 18F Leadership Selection Playbook
+---
+This playbook is intended to capture what we've learned from selecting past 18F Supervisors and Team Leads and help you, future reader, avoid some pitfalls and remember some easy-to-forget nuances. Please feel free to iterate and experiment, but capture the results of those experiments for future selection rounds.
+
+##Week one
+
+Create a nomination form in Google Forms. You can [copy an older example](https://docs.google.com/document/d/1LPQo-yuiabmRGxGtIPKbkq5jevDs8WUPvBuLqB5VQA4/edit>) or just create one from scratch. Typically, this form has been very short and simple with just a few pieces of information:
```

Comment:
small typo in the hyperlink - need to remove the trailing >

Reply:
Resolved.